Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -2323,7 +2324,7 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
-		0000000002D0 /* Podfile */ = {isa = PBXFileReference; explicitFileType = text.script.ruby; includeInIndex = 1; indentWidth = 2; lastKnownFileType = text; name = Podfile; path = ../Podfile; sourceTree = SOURCE_ROOT; tabWidth = 2; xcLanguageSpecificationIdentifier = xcode.lang.ruby; };
+		0000000002D0 /* Podfile */ = {isa = PBXFileReference; explicitFileType = text.script.ruby; includeInIndex = 1; indentWidth = 2; name = Podfile; path = ../Podfile; sourceTree = SOURCE_ROOT; tabWidth = 2; xcLanguageSpecificationIdentifier = xcode.lang.ruby; };
```

Comment:
I wouldn't expect to see changes in the file if you didn't change any native deps. @ashfurrow @alloy can either of you tell if these changes are kosher?

Reply:
Hmm, unsure what would cause these. They don’t look problematic, all seem related to non-critical metadata that Xcode.app updates 🤔 In short, I’m ok with checking these in, but let’s keep an eye on if this keeps changing with each PR.